### PR TITLE
#40 add avatar api

### PR DESCRIPTION
**Pull request by bderevyaga**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,8 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+src/public/*.png
```
Comment (wearethefuture-org) [inline .gitignore]: why only png?

File: src/db/models/relations.ts [hunk 2]
```diff
@@ -120,10 +120,11 @@ export const relations = (db: any): void => {
   //   onDelete: 'CASCADE'
   // });
 
-  // db.users.belongsTo(db.files, {
-  //   as: db.aliases.users.files,
-  //   foreignKey: 'avatarId'
-  // });
+  db.users.belongsTo(db.files, {
```
Comment (wearethefuture-org) [inline src/db/models/relations.ts]: Please Replace higher commented relations

**Comment by wearethefuture-org** [comment]
[#40 ADD /avatar API](https://trello.com/c/OVPtL9ca/40-40-add-avatar-api)

File: src/db/models/relations.ts (hunk 2 above)
Comment (bderevyaga) [reply src/db/models/relations.ts]: Done

File: .gitignore (hunk 1 above)
Comment (bderevyaga) [reply .gitignore]: Done

**Pull request merged**